src/commands/OwnerCommand.js

```diff
@@ -0,0 +1,78 @@
+import Command from "../Command";
+import NpmUtilities from "../NpmUtilities";
+import ConfigUtilities from "../ConfigUtilities";
+import async from "async";
+
+const COMMAND_REQUIRED_ERROR_MESSAGE = "You must specify which command to run, and it must be one of: add, rm";
+const OWNER_NOT_FOUND_ERROR_FRAGMENT = " is not listed as an owner in lerna.json";
+const RM_COMMAND_NAME = "rm";
+const ADD_COMMAND_NAME = "add";
+
+export default class OwnerCommand extends Command {
+  initialize(callback) {
+    this.command = this.input[0];
+    this.ownerName = this.input[1];
+
+    if (!this.command || [RM_COMMAND_NAME, ADD_COMMAND_NAME].indexOf(this.command) < 0) {
+      callback(new Error(COMMAND_REQUIRED_ERROR_MESSAGE));
+      return;
+    }
+
+    callback(null, true);
+  }
+
+  execute(callback) {
+    // Note that we will be mutating object keys, just not the reference.
+    const config = ConfigUtilities.readSync(this.repository.rootPath);
+
+    // There's a lot of legacy lerna.jsons that don't have an owner hash, and we
+    // don't include an owners hash on init.
+    if (!config.owners) {
+      config.owners = [];
+    }
+
+    // We could use array.includes instead, but this has the added benefit of
+    // removing duplicate owners from lerna.json.
+    const ownersSet = new Set(config.owners);
+
+    switch (this.command) {
+    case ADD_COMMAND_NAME:
+      ownersSet.add(this.ownerName);
+      break;
+    case RM_COMMAND_NAME:
+      if (ownersSet.has(this.ownerName)) {
+        ownersSet.delete(this.ownerName);
+      } else {
+        return new callback(new Error(this.ownerName + OWNER_NOT_FOUND_ERROR_FRAGMENT));
```

Comment:
Should this warn and proceed?  Could be trying to remove an owner that was added before owners were stored in lerna config.